src/core/event/EventManager.js

```diff
@@ -0,0 +1,60 @@
+/* global console */
+
+export default class EventManager {
+  constructor(nextHandlers) {
+    this._nextHandlers = nextHandlers;
+  }
+
+  attach(element) {
```

Comment:
So the idea here is that the same handler can be added to multiple elements? 
Something to consider: Is that worth the additional complexity in the API? As opposed to just creating new handlers.